.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for submitting your Pull Request!
+
+Please make sure your PR meets the following requirements:
+
+- [ ] Pull Request title is properly formatted: `[WFLY-XYA] Subject`
```

Comment:
[ and ] around Jira ID are completely redundant and should not be required neither promoted to be used.

Reply:
thanks for the comment, some projects use this notation, also in many WildFly commits

We can have this:
[ ] Pull Request title is properly formatted: `[WFLY-XYZ] Subject` or `WFLY-XYZ Subject`

I personally like [WFLY-XYZ] more :)

Reply:
Sure, but we could also use this opportunity to standardize on the effective form.

Reply:
@kabir ^^^ [ ] or not ?

Reply:
Personally, I don't care one way or another about the brackets. It is very nice of course to have the issue keys in the title. But what I **REALLY** :) care about is having the links in the description so I can click on those